wagtailsharing/tests/settings.py

```diff
@@ -27,7 +29,9 @@
 
 WAGTAIL_APPS = (
     "wagtail.contrib.forms",
-    "wagtail.contrib.modeladmin",
+    "wagtail.contrib.modeladmin"
+    if WAGTAIL_VERSION <= (5, 0)
+    else "wagtail.snippets",
```

Comment:
Based on Dan's comment here: https://github.com/torchbox-forks/wagtail-sharing/pull/3/files#r1368851904
```suggestion
    "wagtail.snippets"
    if WAGTAIL_VERSION >= (5, 1)
    else "wagtail.snippets",
```

Reply:
Thanks @katdom13 If I consider the maintainer has used `<=` may be for their own reasons it would be better to keep that here for now. When we submit this PR to the maintainer I will mention it to them.

The conditions I added in wagtail_hooks are `>=` so there is some inconstancy happening across the code so based on the maintainers feedback that could be adjusted then and we could rebase if changes are needed.